app/routes/profiles/show.js

```diff
@@ -1,7 +1,15 @@
 import Route from '@ember/routing/route';
 
 export default class ShowProjectRoute extends Route {
-  model({ id }) {
-    return this.store.findRecord('profile', id, { reload: true });
+  async model({ id }) {
+    const model = await this.store.findRecord('profile', id, { reload: true });
+    const images = await fetch(`https://api.github.com/repos/nycplanning/labs-waterfront-access-photos/contents/${id}`)
+      .then(d => d.json())
+
+    if (!images.message) {
+      model.set('images', images.map(d => d.download_url))
```

Comment:
FYI, this puts the model in "dirty" state which could have some unexpected consequences in the future. You could avoid this by making it a regular classic property (without the `attr()` decorator), or having a computed on the model that returns this async promise.

Reply:
Yup. When switching from a profile w/ an image to one w/o, `model.images` isn't null.

Reply:
@andycochran hmmm what do you mean? This would only apply to models of the same ID.